samples/server/go-server/api/server/testing/testobjects.go

```diff
@@ -15,16 +15,18 @@
 package testutil
 
 import (
+	"log"
+
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/any"
+	"github.com/grafeas/grafeas/samples/server/go-server/api/server/name"
 	pb "github.com/grafeas/grafeas/v1alpha1/proto"
 	opspb "google.golang.org/genproto/googleapis/longrunning"
-	"log"
 )
 
-func Occurrence(noteName string) *pb.Occurrence {
-	return &pb.Occurrence{
+func Occurrence(noteName string) (*pb.Occurrence, string) {
```

Comment:
I think it would be better to actually have these take a project as an argument so that future tests can easily create objects in separate projects.

Reply:
Sounds good. Same for notes and operations then I guess?

Reply:
Proposed fix in 1c1be49